packages/cli/test/unit/services/webhook.service.test.ts

```diff
@@ -0,0 +1,214 @@
+import { v4 as uuid } from 'uuid';
+import config from '@/config';
+import { mockInstance } from '../../integration/shared/utils/';
+import { WebhookRepository } from '@/databases/repositories';
+import { CacheService } from '@/services/cache.service';
+import { WebhookService } from '@/services/webhook.service';
+import { WebhookEntity } from '@/databases/entities/WebhookEntity';
+
+const createWebhook = (method: string, path: string, webhookId?: string, pathSegments?: number) =>
+	Object.assign(new WebhookEntity(), {
+		method,
+		webhookPath: path,
+		webhookId,
+		pathSegments,
+	}) as WebhookEntity;
```

Comment:
Is there a cleaner way? `WebhookRepository` is being mocked so create gives `undefined`.